app/Notification.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import './_ciq-notification.scss';
+
+import CloseIcon from './icons/ic-close.svg';
+import Warning from './icons/warning.svg';
+import Error from './icons/error.svg';
+import Success from './icons/success.svg';
+import Info from './icons/info.svg';
```

Comment:
charts.binary.com is designed to use smartcharts as an app. Adding these this way would fail cause the environment in `app` does not have `inline-react-svg`. Refer to https://github.com/binary-com/SmartCharts#debugging-npm-package

Reply:
@brucebinary, unfortunately, I can not make this work